src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -129,74 +132,137 @@ export default class BarGraph extends React.Component<IBarGraphProps, {colors: s
             .sort((a, b) => b.caseCount - a.caseCount)
             .slice(0, 20);
 
-        Highcharts.chart(this.chartTarget, {
+        const datasets = _.flattenDeep(cancerTypeStudiesArray.map((cancerStudySet, i) => (
+            cancerStudySet.studies.map((cancerStudy: CancerStudy, j: number) => {
+                const {name, studyId, allSampleCount} = cancerStudy;
+                const max = cancerStudySet.studies.length;
+                const colors = ['1E36BF', '128C47', 'BF2231', '7D1FBF', 'BF7D15'];
+                const color = this.lightenDarkenColor(colors[i%5], (max-j)/max * 120);
+                return {
+                    borderColor: '#F1F6FE',
+                    backgroundColor: color,
+                    borderWidth: 1,
+                    label: name,
+                    total: cancerStudySet.caseCount,
+                    studyId,
+                    data: i === 0 ? [allSampleCount] : [...Array(i).fill(0), allSampleCount]
+                };}
+            ))
+        ));
+
+        const data = {
+            labels: cancerTypeStudiesArray.slice(0, 20).map(cancer => (cancer.shortName)),
+            datasets
+        };
 
-            chart: {type: 'bar', backgroundColor:'' },
+        const options = {
+            onClick: function(e: Event) {
+                if (this.getElementAtEvent(e)[0]) {
+                    const {studyId} = datasets[this.getElementAtEvent(e)[0]._datasetIndex];
+                    window.location.href = 'study?id=' + studyId + '#summary';
+                }
+            },
+            responsive: true,
+            tooltips: {
+                enabled: false,
+                mode: 'nearest',
+                custom(tooltipModel: any) {
+                    // Tooltip Element
```

Comment:
i'm interested to know, when tooltip fails to show up (you demo'd this the other day) is this function just not firing or does it fire but with somehow faulty arguments?